fhir-server-test/src/test/java/com/ibm/fhir/server/test/SearchTest.java

```diff
@@ -154,52 +167,52 @@ public void testCreatePersonWithManyIdentifiers() throws Exception {
         TestUtil.assertResourceEquals(person, responsePerson);
     }
 
-    // Uncomment in order to test the limits when using the 'normalized' model with
-    // Db2
-    // @Test(groups = { "server-search" })
-    // public void testCreatePersonWithTooManyLinks() throws Exception {
-    // // Build a new Person with a single identifier and then call call 'create-on-update'.
-    // // This will ensure that the search parameters are added to the cache before we insert one with a thousand of
-    // them.
-    // String identifier = UUID.randomUUID().toString();
-    // String patientRef = "Patient/" + identifier;
-    // Person person = createPersonWithIdentifierAndLink(identifier, patientRef);
+    //    // Uncomment in order to test the limits when using the 'normalized' model with
+    //    // Db2
+    //    @Test(groups = { "server-search" })
+    //    public void testCreatePersonWithTooManyLinks() throws Exception {
+    //        // Build a new Person with a single identifier and then call call 'create-on-update'.
+    //        // This will ensure that the search parameters are added to the cache before we insert one with a thousand of
+    //        //them.
+    //        String identifier = UUID.randomUUID().toString();
+    //        String patientRef = "Patient/" + identifier;
+    //        Person person = createPersonWithIdentifierAndLink(identifier, patientRef);
```

Comment:
Not a fan of the double-commenting.
We discussed that these limits are no longer valid and so I think the tests should be remove.
If we add limits like these back in later, we can introduce the tests then.

Reply:
sounds good, will remove these 2 already commented out tests.

Reply:
done